### DISCUSSION TITLE
字体颜色

### DISCUSSION THREAD

**Discussion by Soywei00** [openneno/neno#15] [Ideas]
字体颜色可以使用黑色吗？感觉粉红有点违和……

**Comment by Mran** [comment]
其他方面呢，如果设计一套可以自定义主题的功能会不会更好？使用css进行配置。

**Reply by Soywei00** [reply]
从功能来说，如果可以自定义主题（编辑器设置）的话当然更好。
但是从卡片笔记这个角度出发的话，我建议直接上几套颜色主题，毕竟flomo这种类型的初衷就是快速记录想法，折腾主题啥的是notion obsidian那些重度笔记软件的事情，而且CSS虽然简单，但毕竟需要花一点点时间学习，这可能会劝退一部分人。
主题的元素只要有字体和颜色就好，可以使用开源字体，比如霞鹜文楷（GitHub上有开源）、思源宋体这些，字体颜色就设置常见的黑、黄、红、绿这几种就好，不超过5种吧。
这是我的一点建议。

**Reply by Mran** [reply]
你说的有道理，我想想怎么实现这个方案，感谢你的提议。

**Comment by Soywei00** [comment]
还有一个反馈，好像utools插件还不支持gitee？<html>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
</head>
<body>
<br/><br/><br/>        <a href="https://maas.mail.163.com/dashi-web-extend/html/proSignature.html?ftlId=1&name=soywei00&uid=soywei00%40outlook.com&iconUrl=https%3A%2F%2Fmail-online.nosdn.127.net%2Fqiyelogo%2FdefaultAvatar.png&items=%5B%22%E9%82%AE%E7%AE%B1%EF%BC%9Asoywei00%40outlook.com%22%5D" style="display:block;background:#fff; max-width: 400px; _width: 400px;padding:15px 0 10px 0;text-decoration: none; outline:none;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:none !important;text-size-adjust:none !important;">
            <table cellpadding="0" style='width: 100%; max-width: 100%; table-layout: fixed; border-collapse: collapse;color: #9b9ea1;font-size: 14px;line-height:1.3;-webkit-text-size-adjust:none !important;text-size-adjust:none !important;'>
                <tbody style="font-family: 'PingFang SC', 'Hiragino Sans GB','WenQuanYi Micro Hei', 'Microsoft Yahei', '微软雅黑', verdana !important; word-wrap:break-word; word-break:break-all;-webkit-text-size-adjust:none !important;text-size-adjust:none !important;">
                    <tr>
                            <td width="38" style="padding:0; box-sizing: border-box; width: 38px;">
                                <img width="38" height="38" style="vertical-align:middle; width: 38px; height: 38px; border-radius:50%;" src="https://mail-online.nosdn.127.net/qiyelogo/defaultAvatar.png" />
                            </td>
                            <td style='padding: 0 0 0 10px; color: #31353b;'>
                                <div style="font-size: 16px;font-weight:bold; width:100%; white-space: nowrap; overflow:hidden;text-overflow: ellipsis;">soywei00</div>
                            </td>
                    </tr>
                        <tr width="100%" style="font-size: 14px !important; width: 100%;">
                            <td colspan='2' style="padding:10px 0 0 0; font-size:14px !important; width: 100%;">
                                    <div style="width: 100%;font-size: 14px ***@***.***</div>
                            </td>
                        </tr>
                </tbody>
            </table>
        </a><br/><br/><br/><div class="ntes-mailmaster-quote" style="padding-top: 1px; padding-bottom: 1px" >
    <div style=" margin-top: 2em; margin-bottom: 1em; font-size: 14px; line-height: 1.25; color: #78787a; " >---- 回复的原邮件 ----</div>
    <div style=" margin-bottom: 1em; font-size: 12px; line-height: 1.25; color: #232324; padding: 0.5em 0.25em; border-radius: 0.67em; background-color: #f0f0f0; " >
        <table width="100%" cellpadding="0" cellspacing="9" border="0">
            <tr>
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    发件人
                </td>
                <td valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    <a class="mail-from" style="text-decoration:none;color:#0886e8;" ***@***.******@***.***&gt;</a>
                </td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    日期
                </td>
                <td class="mail-date" valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    2022年02月14日 11:57
                </td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    收件人
                </td>
                <td valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    <a class="mail-to" style="text-decoration:none;color:#0886e8;" ***@***.******@***.***&gt;</a>
                </td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    抄送至
                </td>
                <td valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    <a class="mail-cc" style="text-decoration:none;color:#0886e8;" ***@***.******@***.***&gt;</a>、<a class="mail-cc" style="text-decoration:none;color:#0886e8;" ***@***.******@***.***&gt;</a>
                </td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    主题
                </td>
                <td class="mail-subject" valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    Re: [openneno/neno] 字体颜色 (Discussion #15)
                </td>
            </tr>
        </table>
    </div>
    <div><meta http-equiv="Content-Type" content="text/html; charset=utf-8"><p></p>

<p dir="auto">你说的有道理，我想想怎么实现这个方案，感谢你的提议。</p>



<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">—<br>Reply to this email directly, <a href="https://github.com/openneno/neno/discussions/15#discussioncomment-2170378">view it on GitHub</a>, or <a href="https://github.com/notifications/unsubscribe-auth/AD7WUBM5YJ4YCRF2QNBF5TTU3B4ULANCNFSM5OKF7B3Q">unsubscribe</a>.<br>Triage notifications on the go with GitHub Mobile for <a href="https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675">iOS</a> or <a href="https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub">Android</a>.

<br>You are receiving this because you authored the thread.<img src="https://github.com/notifications/beacon/AD7WUBPVKFGAKMA52VZPUXDU3B4ULA5CNFSM5OKF7B32YY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZGOAAQR4CQ.gif" height="1" width="1" alt=""><span style="color: transparent; font-size: 0; display: none; visibility: hidden; overflow: hidden; opacity: 0; width: 0; height: 0; max-width: 0; max-height: 0; mso-hide: all">Message ID: <span>&lt;openneno/neno/repo-discussions/15/comments/2170378</span><span>@</span><span>github</span><span>.</span><span>com&gt;</span></span></p>

<script type="application/ld+json">[

{

***@***.***": "http://schema.org",

***@***.***": "EmailMessage",

"potentialAction": {

***@***.***": "ViewAction",

"target": "https://github.com/openneno/neno/discussions/15#discussioncomment-2170378",

"url": "https://github.com/openneno/neno/discussions/15#discussioncomment-2170378",

"name": "View Discussion"

},

"description": "View this Discussion on GitHub",

"publisher": {

***@***.***": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]</script></div>
</div>
</body>
</html>

**Reply by Mran** [reply]
是的，这个改动还没同步到utool上去，目前的用户配置输入有点复杂了，正在设计一个更好的用户配置输入方案。
